tests/vmi_configuration_test.go

```diff
@@ -1670,17 +1672,20 @@ var _ = Describe("[sig-compute]Configurations", func() {
 		})
 
 		Context("[Serial]using defaultRuntimeClass configuration", func() {
-			runtimeClassName := "custom-runtime-class"
+			var runtimeClassName string
 
 			BeforeEach(func() {
+				// use random storage class to avoid collisions with cleanup where a
```

Comment:
Shouldn't it be *[runtime](https://kubernetes.io/docs/concepts/containers/runtime-class/)* class instead of *[storage](https://kubernetes.io/docs/concepts/storage/storage-classes/)* class? Same question about the PR description and the commit message.

Reply:
ha, definitely

Reply:
Fixed.